src/main/java/seedu/badmaths/NotesFileContentManager.java

```diff
@@ -0,0 +1,34 @@
+package seedu.badmaths;
+
+import java.util.ArrayList;
+
+public class NotesFileContentManager {
+    public static String fileContent(ArrayList<Note> notes) {
+        StringBuilder content = new StringBuilder();
+        for(Note note : notes) {
+            //priority
+            NotePriority.Priority priority = note.getPriority();
```

Comment:
Consider refactoring line 10 to 12 to a method called `parsePriorityInfoToContent(Note note)`. You can do the same for status, count, text to make your code neater.